tgui/packages/tgui/interfaces/LibraryConsole.js

```diff
@@ -0,0 +1,988 @@
+    context, 'ModalUpload', '');
+
+  const display_category = uploadCategory || default_category;
+  return (
+    <Modal>
+      <Box fontSize="20px" pb={1}>
+        Are you sure you want to upload this book to the database?
+      </Box>
+      <LabeledList>
+        <LabeledList.Item label="Category">
+          <Dropdown
+            options={upload_categories}
+            selected={display_category}
+            onSelected={(value) => setUploadCategory(value)} />
+        </LabeledList.Item>
+      </LabeledList>
+      <Stack justify="center" align="center" pt={2}>
+        <Stack.Item>
+          <Button
+            disabled={!can_db_request}
+            icon="upload"
+            content="Upload To DB"
+            fontSize="18px"
+            color="good"
+            onClick={() => {
+              setUploadToDB(false);
+              act('upload', {
+                category: display_category,
+              });
+            }}
+            lineHeight={2} />
+        </Stack.Item>
+        <Stack.Item>
+          <Button
+            icon="times"
+            content="Return"
+            fontSize="18px"
+            color="bad"
+            onClick={() => setUploadToDB(false)}
+            lineHeight={2} />
+        </Stack.Item>
+      </Stack>
+    </Modal>
+  );
+};
+
+export const Print = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    deity,
+    religion,
+    bible_name,
+    bible_sprite,
+    posters,
+  } = data;
+  const [selectedPoster, setSelectedPoster] = useLocalState(
+    context, 'selected_poster', posters[0]);
+
+  return (
+    <Stack height="100%">
+      <Stack.Item grow>
+        <Stack vertical height="100%">
+          <Stack.Item height="100%">
+            <Section fill scrollable>
+              {posters.map(poster => (
+                <div
+                  key={poster}
+                  title={poster}
+                  className={classes([
+                    'Button',
+                    'Button--fluid',
+                    'Button--color--transparent',
+                    'Button--ellipsis',
+                    selectedPoster
+                    && poster === selectedPoster
+                    && 'Button--selected',
+                  ])}
+                  onClick={() => setSelectedPoster(poster)} >
+                  {poster}
+                </div>
+              ))}
+            </Section>
+          </Stack.Item>
+          <Stack.Item>
+            <Button
+              fluid
+              icon="scroll"
+              content="Poster"
+              fontSize="30px"
+              lineHeight={2}
+              onClick={() => act('print_poster', {
+                poster_name: selectedPoster,
+              })} />
+          </Stack.Item>
+        </Stack>
+      </Stack.Item>
+      <Stack.Item grow>
+        <Stack vertical height="100%">
+          <Stack.Item grow>
+            <Stack vertical height="100%">
```

Comment:
Unnecessary stack, unwrap it.

Reply:
I remember there being a reason for this, might have just been using stack wrong tho

Reply:
Ah yeah uh.
I want the stack item to stick to the bottom of the window, if I just remove the second stack that breaks
![image](https://user-images.githubusercontent.com/58055496/157172415-f9f3bb4b-ec6e-4458-96e4-5732786498c5.png)

I think I could use mt="auto" for this, but that breaks when I resize the window, which I'm not a huge fan of

![image](https://user-images.githubusercontent.com/58055496/157172245-9791cf04-1f89-4622-b11a-cd518337552b.png)

Am I doing something wrong here? or is that overlap behavior just kinda how it should work

Reply:
You're doing something wrong, but ok.

Reply:
Made better. Think I can go further with it, will work on it tomorrow